rfcs/block-component.md

```diff
@@ -67,14 +120,3 @@ The reality is you often want to apply multiple of these concerns to one element
 Having multiple elements can complicate things like layout (flexbox children, etc), as well as just needlessly increasing the amount of DOM the client has to deal with.
 
 Combining most things into a single `Block` component allows you to keep things as a single element, and generally has less cognitive overload due to fewer react elements. (maybe there's a happy middle-ground though, for example a `Block` and `Text` component?)
-
```

Comment:
Do we normally want to leave sections like this in an RFC after implementation? Willing to leave it if it's normal.

Reply:
Probably fine to Nuke this, I'd imagine we can also go back to the original PR if we want to look at the related discussion.